This patch solves the problem that ipc() multiplexer calls could not be
filtered out in strace by "-e target='!ipc'". The calls would still occur in
strace's log output. See our upstream inquiry at

https://sourceforge.net/mailarchive/forum.php?thread_name=20111125185651.GA18385%40altlinux.org&forum_name=strace-devel

From: Dmitry V. Levin <ldv@altlinux.org>
Date: Fri, 25 Nov 2011 19:27:23 +0000 (+0000)
Subject: Add TRACE_IPC flag to sys_ipc syscall entries
X-Git-Url: http://strace.git.sourceforge.net/git/gitweb.cgi?p=strace%2Fstrace;a=commitdiff_plain;h=454dcd458aa0f8ebe6838c1835d779739a9cca77

Add TRACE_IPC flag to sys_ipc syscall entries

* linux/mips/syscallent.h: Add TI flag to sys_ipc entry.
---

diff --git linux/mips/syscallent.h linux/mips/syscallent.h
index 5b42b7e..f1e927c 100644
--- linux/mips/syscallent.h	2011-11-26 12:56:01+01:00
+++ linux/mips/syscallent.h	2011-11-26 12:56:01+01:00
@@ -4119,7 +4119,7 @@
 	{ 4,	TP,	sys_wait4,		"wait4"		}, /* 4114 */
 	{ 1,	TF,	sys_swapoff,		"swapoff"	}, /* 4115 */
 	{ 1,	0,	sys_sysinfo,		"sysinfo"	}, /* 4116 */
-	{ 6,	0,	sys_ipc,		"ipc"		}, /* 4117 */
+	{ 6,	TI,	sys_ipc,		"ipc"		}, /* 4117 */
 	{ 1,	TD,	sys_fsync,		"fsync"		}, /* 4118 */
 	{ 1,	TS,	sys_sigreturn,		"sigreturn"	}, /* 4119 */
 	{ 5,	TP,	sys_clone,		"clone"		}, /* 4120 */
